.github/workflows/ci.yml

```diff
@@ -221,6 +221,31 @@ jobs:
         chmod +x endless-sky
         ./tests/test_parse.sh ./endless-sky
 
+
+  # This section is for tests on ubuntu 16 (and on ubuntu 20).
+  # The tests with Xvfb in this section work fine on ubuntu 16 and on Ubuntu 20, but they don't work on Ubuntu 18.
+  test_ubuntu16:
```

Comment:
perhaps we should rename this to `autotest_ubuntu16`, and the others to `parse_*`

Reply:
Good idea, done.
I also re-ordered them so that all the parse-tests are together again.